composer.json

```diff
@@ -38,24 +38,20 @@
     "scripts": {
         "check": [
             "@composer-validate",
-            "@lint",
```

Comment:
Why remove the lint step? Having PHP + composer.lock linting has proven very usefull in the past.

Reply:
There was no linter in this project

Reply:
@MKodde 
Regarding this and the previous comment. Maybe it is a good time to overthink how to prevent these differences across projects. If we are really committed to this, we should create a reference project and All projects need to comply. 

Now I've seen defferences in details on a per project basis, never knowing what is actually the required standard.

Reply:
We have a standard set of tools in our QA toolbelt. We look at every project and set up the applicable ones for that project. But a linting tool should fit any project right.